macos/Sources/Ghostty/Package.swift

```diff
@@ -257,6 +257,9 @@ extension Notification.Name {
     /// Ring the bell
     static let ghosttyBellDidRing = Notification.Name("com.mitchellh.ghostty.ghosttyBellDidRing")
     static let ghosttyCommandPaletteDidToggle = Notification.Name("com.mitchellh.ghostty.commandPaletteDidToggle")
+
+    /// Toggle maximize of current window
+    static let ghosttyToggleMaximize = Notification.Name("com.mitchellh.ghostty.toggleMaximize")
```

Comment:
I try to follow AppKit's notification naming convention of roughly a noun phrase followed by a past participle adjective. i.e. `[Subject] [Event (past participle)]`.

So, we should do `ghosttyFullscreenDidToggle`. 

There's an inconsistency here because the notifications were never fully converted to this style (we should do that in a future PR) but for new stuff we should follow it.